### DISCUSSION TITLE
Not working with Android grabber

### DISCUSSION THREAD

**Discussion by haze1986** [awawa-dev/HyperHDR#962] [General]
Hi all, I have no success in getting hyperHDR working with hyperion-android-grabber despite setting the priority between 100 to 199. 
https://github.com/abrenoch/hyperion-android-grabber
I have no issues with connections using mostly the same settings in Hyperion.

Can I confirm it is supported for the latest version?

**Comment by rice1204** [comment]
I'm new to HyperHDR myself and just got it running with Hyperion android grabber.  I'm not sure if this is the proper method, but what I had to do to get it running was this:

HyperHDR -> settings -> Advanced -> Network Services -> Activate Protocol Buffers Server

Of course, make sure the port matches what hyperion is expecting. You can also telnet into your HyperHDR install on that port to verify it's running correctly and rule out any connectivity problems.
